src/like-service/routes/web.php

```diff
@@ -13,10 +13,10 @@
 |
 */
 
-Route::get('/', function () {
+Route::get('/like-service/', function () {
```

Comment:
[DISCUSS] I think here we should also use the env vars (base path)

Reply:
env vars are now used